lang/fpc/Portfile

```diff
@@ -29,7 +29,8 @@ set boot_v          3.0.4
 
 master_sites        sourceforge:freepascal:main \
                     https://downloads.freepascal.org/fpc/dist/${boot_v}/bootstrap/:bootstrap \
-                    https://downloads.freepascal.org/fpc/dist/${boot_v}/powerpc-macosx/:bootstrap
+                    https://downloads.freepascal.org/fpc/dist/${boot_v}/powerpc-macosx/:bootstrap \
+                    https://downloads.freepascal.org/fpc/dist/${version}/i386-macosx/:bootstrap
```

Comment:
@kamischi Do you use `version` here?

Reply:
yes. In lucky cases, this may save the 3-fold cycling.